src/ritika/assignments_1to20/Asgmt_04_Form.java

```diff
@@ -0,0 +1,111 @@
+package ritika.assignments_1to20;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import technoCredits.base.PredefinedActions;
+
+/*Assignment - 4 [Form : /resources/forms/First_Form_4]
+
+a) Revert the selection of 'Gender' radio button.
+Example: If 'Male' is preselected then select 'Female'.
+If 'Female' is preselected then select 'Male'.
+
+b) Revert the selection of checkboxes under 'language Wanted to learn'
+Example: Whatever language is selected in checkboxes, please uncheck those
+& Select unchecked checkboxes
+
+c) Select 'Second Last' index from 'I am living in' dropdown and print value of selected option.
+Note: This is dynamic dropdown. Options may get changed in future. So don't search it by name.
+We want second last index in any situation
+
+d) Validate 'Total Year's of Experience In Industry' radio button.
+Print message like below based on preselected experience radio button.
+1. Student -> print 'Beginner'
+2. 1-3 -> print 'Upper Beginner'
+3. 3-6 -> print 'Intermediate'
+4. 6-10 -> print 'Advanced'
+5. Above 10 -> print 'Expert'
+
+e) Validate Reset functionality of form.
+1. Type something in 'First Name'. Validate text of this field.
+2. Click on 'Reset form in same tab' button.
+3. Validate text of 'First Name' field again. It should be empty or null.*/
+
+public class Asgmt_04_Form {
+
+	static void formOperations(WebDriver driver) {
+		driver.manage().timeouts().implicitlyWait(3, TimeUnit.SECONDS);
+		System.out.println("Step - Revert the selection of 'Gender' radio button");
+		List<WebElement> list = driver.findElements(By.xpath("//input[@name=\"gender\"]"));
```

Comment:
you should write xpath like below

//input[@name='gender'], your xpath will work also.